### ISSUE TITLE
Need summary stats added back to the top of the validation results

### ISSUE THREAD

**Issue by GuardRex** [validator/validator#162]
The old checker totaled and reported the number of errors and warnings and reported the totals at the top of the results. We need that functionality back in the tool.

**Comment by sideshowbarker** [comment]
> The old checker totaled and reported the number of errors and warnings and reported the totals at the top of the results. We need that functionality back in the tool.

You can get that information now by using the **Message Filtering** button in the Web UI.

**Comment by GuardRex** [comment]
@sideshowbarker Thanks ... reading the page carefully wasn't something I did obviously ... I guess I was just used to the old days (the previous checker) and the stats are so openly stated in that prior version. I thought it was removed in this checker. The fact that the text there is grayed-down makes it even easier to miss, because when people read webpages, they probably discount text that's grayed-down in value as very unimportant information.

Suggestion: Incorporate text into the button to make it really obvious that you'll get the page stats there: "Message Filtering and Page Statistics" as button text would be quickly picked up.
